cmake/configs/default.cmake

```
@@ -257,7 +257,7 @@ hunter_default_version(dfdutils VERSION 0.0.0-659a739-p1)
 hunter_default_version(dlib VERSION 19.17-p0)
 hunter_default_version(dlpack VERSION 0.2-0acb731)
 hunter_default_version(dmlc-core VERSION 0.3-3943914-p0)
-hunter_default_version(doctest VERSION 2.4.4)
+hunter_default_version(doctest VERSION v2.4.9)
```

the other packages don't use the leading `v` in the version number. The `v` isn't technically part of the version number generally

```suggestion
hunter_default_version(doctest VERSION 2.4.9)
```

---

Some other packages seem to use it so I assumed it was dependent on the versioning scheme used by the project proper, thanks!